UMI3D-SDK/Assets/ClientDevelopmentKit/UserCapture/Runtime/UserAvatar.cs

```diff
@@ -177,11 +188,16 @@ Transform InspectBoundRigs(BoneBindingDto dto)
 
         IEnumerator WaitForRig(BoneBindingDto dto, UMI3DClientUserTrackingBone bone)
         {
-            UMI3DNodeInstance node = UMI3DEnvironmentLoader.GetNode(dto.objectId);
+            UMI3DNodeInstance node;
+            var wait = new WaitForFixedUpdate();
+
+            while ((node = UMI3DEnvironmentLoader.GetNode(dto.objectId)) == null)
+            {
+                yield return wait;
+            }
```

Comment:
Prefer Unity's syntax: 
`yield return new WaitWhile(() => (node = UMI3DEnvironmentLoader.GetNode(dto.objectId)) == null )`

Reply:
What is the usage of wait ?

Reply:
Note that `WaitWhile` will be evaluate each frame whereas `WaitForFixedUpdate` will skip some.